src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -778,8 +779,13 @@ private String formatDestMount(String destMountString, File srcMountFile) {
         return destMountString;
     }
 
-    private File createTempDockerfile(List<String> dockerfileLines) throws PluginExecutionException {
+    private File prepareTempDockerfile(File dockerfile) throws PluginExecutionException {
         // Create a temp Dockerfile to build image from
+
+        List<String> dockerfileLines = readDockerfile(dockerfile);
+        dockerfileLines = removeWarFileLines(dockerfileLines);
```

Comment:
This is a nit, but there is no need to pass back the List from these methods (removeWarFileLines and removeCopyLines). It was passed in and modified.

Reply:
Fixed